RichEditorView/Assets/editor/rich_editor.js

```diff
@@ -231,8 +237,10 @@ RE.insertImage = function(url, alt) {
     var img = document.createElement('img');
     img.setAttribute("src", url);
     img.setAttribute("alt", alt);
+    img.setAttribute("width", "150");
```

Comment:
humm... this does not seem right: hardcoded. If you really need to do this, can you do it with css?

Reply:
ok, got those variables added as parameters on the JS script.... 
![Screen Shot 2019-10-03 at 6 10 43 PM](https://user-images.githubusercontent.com/50113568/66170288-4ba96e80-e609-11e9-8851-c46ac4659477.png)